bayes_opt/domain_reduction.py

```diff
@@ -45,24 +47,26 @@ class SequentialDomainReductionTransformer(DomainTransformer):
         Zooming parameter used to shrink the region of interest.
 
     minimum_window : float or np.ndarray or dict, default=0.0
-        Minimum window size for each parameter. If a float is provided, the same value is used for all parameters.
+        Minimum window size for each parameter. If a float is provided,
+        the same value is used for all parameters.
     """
 
     def __init__(
         self,
         gamma_osc: float = 0.7,
         gamma_pan: float = 1.0,
         eta: float = 0.9,
-        minimum_window: Optional[Union[List[float], float, Dict[str, float]]] = 0.0
+        minimum_window: list[float] | float | dict[str, float] | None = 0.0,
```

Comment:
This could also be a `np.ndarray`, I think

Reply:
Fixed some `type hints` that were being written while applying the `ruff` rule.
If you want to add `ndarray`, I'll add it.